README.md

```diff
@@ -121,6 +121,52 @@ Records can be destroyed using the `destroy` method on a table:
 @table.destroy(record)
 ```
 
+## Command Line Tool
+
+This gem is include a very simple command line tool which can show basic functionality of service.
+
+```
+$ airtable
+Usage: airtable [options]
+
+Common options:
+    -k, --api_key=KEY                Airtable API key
+    -t, --table NAME                 Table Name
+    -b, --base BASE_ID               Base ID
+    -r, --record RECORD_ID           Record ID
+    -f, --field FIELD_NAME           Field name to update or read
+    -v, --value VALUE                Field value for update
+
+Supported Operations:
+	Get Record (if only RECORD_ID provided)
+	Get Field (if RECORD_ID and FIELD_ID are provided)
+	Update Field (if RECORD_ID, FIELD_ID and VALUE are provided)
```

Comment:
Lets make the operation explicit:

`airtable get -b Base -t table`
`airtable get -b Base -t table -f FIELD_NAME`

`airtable update -b Base -t table -r RECORD_ID -f FIELD_NAME -v newValue`

Reply:
Sure. Will start working on it shortly.

Reply:
Done